init/atboot.pl

```diff
@@ -7,6 +7,8 @@
 $product = $ARGV[0] || "webmin";
 $ucproduct = ucfirst($product);
 
+$< == 0 || die "updateboot.pl must be run as root";
```

Comment:
Shouldn't this be atboot.pl ?

Reply:
Yes, it's for `atboot.pl` but I forgot to change the name in die.

Fixed.